draggable.js

```diff
@@ -5,54 +5,83 @@
   else this[moduleName] = definition();
 
 })('draggable', function definition() {
-  function addEventListener(element, eventName, handler) {
-                if (element.addEventListener) {
-                    element.addEventListener(eventName, handler, false);
-                } else if (element.attachEvent) {
-                    element.attachEvent('on' + eventName, handler);
+  var addEventListener = ( function (w) {
+                if (w.addEventListener) {
+                    return function (element, eventName, handler){
+                      element.addEventListener(eventName, handler, false);
+                    }
+                } else if (w.attachEvent) {
+                    return function (element, eventName, handler){
+                      element.attachEvent('on' + eventName, handler);
+                    } 
                 } else {
+                  return function (element, eventName, handler){
                     element['on' + eventName] = handler;
+                  }
                 }
-            }
-  function removeEventListener(element, eventName, handler) {
-      if (element.removeEventListener) {
+            })(window)
+  ,removeEventListener = ( function(w) {
+      if (w.removeEventListener) {
+        return function (element, eventName, handler){
           element.removeEventListener(eventName, handler, false);
-      } else if (element.detachEvent) {
+        }
+      } else if (w.detachEvent) {
```

Comment:
the same here. Only needed for IE8 and below. btw microsoft stop support for IE8 on January 12, 2016